Eastern-Research-Group/csb-rebate-forms-app
# Feature/remove formio pseudo api keys

## Related Issues:
* CSBAPP-276

## Main Changes:
Removes no longer necessary `FORMIO_NCES_API_KEY` and `FORMIO_DUPLICATES_API_KEY` env variables, as the formio/nces API endpoint requires authentication, and the bap/duplicates API endpoint will as well, after the Formio deduplication prototype form's functionality gets merged into a future Formio form (likely the 2024 FRF) – for now that endpoint is CORS enabled to support testing and development of the prototype form, but once that logic gets merged into the wrapper app serving the form, enabling CORS should be removed.

## Steps To Test:
1. Navigate to a 2023 FRF, and ensure the NCES fields still populate correctly.
2. Test the prototype Formio deduplication prototype and ensure API calls continue to work as expected.

## TODO:
- [ ] After development of the deduplication prototype form is complete (and before the v5 release), disable CORS from bap/duplicates API endpoint and require authentication for route
